### PR TITLE
added gif to parcoords docs

### PR DESCRIPTION
**Pull request by Kully**
Regarding Issue: https://github.com/plotly/documentation/issues/824

### CODE REVIEW

**Comment by cldougl** [comment]
great 💃 
you could also add this gif to the s3 bucket (like the thumbnails) so that you can reference the hosted link for all of the parcoord doc pages instead of adding it locally for each

**Comment by Kully** [comment]
> you could also add this gif to the s3 bucket

Yes I can do that, but is it a better idea to make a GIF folder instead and start putting all non-thumbnail GIFs in there?

Also, I want this PR to change all the docs at once so I changed the name to reflect that

**Comment by cldougl** [comment]
yep there's an images folder at the same level as thumbnails: https://s3.console.aws.amazon.com/s3/buckets/plotly-tutorials/plotly-documentation/?region=us-east-1&tab=overview
that would be a good spot

**Comment by Kully** [comment]
@cldougl ready for a check

**Comment by cldougl** [comment]
cool - I think some extra files snuck in:
https://github.com/plotly/documentation/blob/99ae5979fb22aea4768bef2961d3ae28f189ee2d/_posts/r/scientific/.Rapp.history
and the gif (since it's hosted you don't need to add it locally here)

**Comment by cldougl** [comment]
also I don't think the js doc changed worked but you could add it via markdown: if you checkout the readme here: https://github.com/plotly/documentation/blob/source-design-merge/_posts/plotly_js/README.md you can see how you can add markdown content or lmk if you have any questions about that.

File: _posts/plotly_js/scientific/parcoords/2017-10-30-parcoords_gif.html [hunk 1]
```diff
@@ -4,13 +4,11 @@
...
+    Parallel coordinates are richly interactive by default. Drag the lines along the axes to filter regions and drag the axis names across the plot to rearrange variables.
 
-<BODY>
-  <IMG SRC="https://s3-us-west-1.amazonaws.com/plotly-tutorials/plotly-documentation/images/js_parcoords_ex1.gif">
-</BODY>
\ No newline at end of file
+---
+<body>
+  <img src="https://s3-us-west-1.amazonaws.com/plotly-tutorials/plotly-documentation/images/js_parcoords_ex1.gif">
+</body>
```
Comment (Kully) [inline _posts/plotly_js/scientific/parcoords/2017-10-30-parcoords_gif.html]: @cldougl I'm not seeing the GIF when I `jekyll serve` whereas I was before. Does this look correct to you? Does `markdown_content` only work in the .index file?

Comment (cldougl) [reply _posts/plotly_js/scientific/parcoords/2017-10-30-parcoords_gif.html]: can you show me how you were seeing it before? I wasn't which is why I suggested this.
The markdown content would be included in the example you want to associated this gif with
like: https://github.com/plotly/documentation/blob/source-design-merge/_posts/plotly_js/animations/animations/2016-09-15-animations-animating-sequences-of-frames.html#L9-L12

Comment (Kully) [reply _posts/plotly_js/scientific/parcoords/2017-10-30-parcoords_gif.html]: okay, it's fixed. This is the output on the page now:

screen shot 2017-10-31 at 5 17 15 pm

File: _posts/plotly_js/scientific/parcoords/2017-03-06-adding_dimensions.html [hunk 2]
```diff
@@ -6,7 +6,9 @@
 order: 1
 sitemap: false
 arrangement: horizontal
-description: 
+description:
+markdown_content: |
+    Parallel coordinates are richly interactive by default. Drag the lines along the axes to filter regions and drag the axis names across the plot to rearrange variables:
```
Comment (cldougl) [inline _posts/plotly_js/scientific/parcoords/2017-03-06-adding_dimensions.html]: @Kully you can actually put the gif in the markdown too like: 

```
---
name: Adding Dimensions
plot_url: https://codepen.io/plotly/embed/ZepaxM?height=500&theme-id=15263&default-tab=result
language: plotly_js
suite: parcoords
order: 1
sitemap: false
arrangement: horizontal
description:
markdown_content: |
    Parallel coordinates are richly interactive by default. Drag the lines along the axes to filter regions and drag the axis names across the plot to rearrange variables: ![IPython terminal](https://s3-us-west-1.amazonaws.com/plotly-tutorials/plotly-documentation/images/js_parcoords_ex1.gif)
---

var trace = {
  type: 'parcoords',
  line: {
    color: 'blue'
  },

  dimensions: [{
    range: [1, 5],
    constraintrange: [1, 2],
    label: 'A',
    values: [1,4]
  }, {
    range: [1,5],
    label: 'B',
    values: [3,1.5],
    tickvals: [1.5,3,4.5]
  }, {
    range: [1, 5],
    label: 'C',
    values: [2,4],
    tickvals: [1,2,4,5],
    ticktext: ['text 1','text 2','text 4','text 5']
  }, {
    range: [1, 5],
    label: 'D',
    values: [4,2]
  }]
};

var data = [trace]

Plotly.plot('graphDiv', data);
```

that way the gif is close to the text describing it and the code is a bit simpler

Comment (cldougl) [reply _posts/plotly_js/scientific/parcoords/2017-03-06-adding_dimensions.html]: otherwise this looks good to go :D

**Comment by Kully** [comment]
@cldougl Done! Can I get a 💃 ?

**Comment by cldougl** [comment]
yep 💃 
(no problem for me but just fyi you have some table files in here too)

**Comment by Kully** [comment]
> yep 💃
> (no problem for me but just fyi you have some table files in here too)

I thought I deleted. Oh well, I'll be needing the table files for the next PR/issue anyways

**Pull request merged**